components/collection/activity/events/Events.vue

```diff
@@ -96,15 +96,17 @@ const filteredEvents = computed(() => {
   const filterByVerifiedIdentity = isOnlyVerifiedUsersFilterActive.value
 
   return props.events.filter((event) => {
-    const active = getEventAddresses(event)
-      .map(getIdentityId)
-      .some((x) => identityIds.includes(x))
-
-    if (!isAnyEventTypeFilterActive() && filterByVerifiedIdentity) {
-      return active
+    const isActiveEvent = filterByVerifiedIdentity
+      ? getEventAddresses(event)
+          .map(getIdentityId)
+          .some((x) => identityIds.includes(x))
+      : true
```

Comment:
hmmm.. always return true if false
any way to refactor it for better readability?

Reply:
back log please